File: add.c
#include <stdio.h>
int main() {
    int a, b;
    printf("Enter two integers: ");
    if (scanf("%d %d", &a, &b) != 2) {
        printf("Invalid input. Please enter integers only.\n");
        return 1; 
    }
    int sum = a + b;
    printf("Sum = %d\n", sum);
    return 0;
}

File: add.java
Javaimport java.util.Scanner;
public class SumProgram {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        try {
            System.out.print("Enter first number: ");
            if (!sc.hasNextDouble()) {
                System.out.println("Invalid input. Please enter a number.");
                return;
            }
            double num1 = sc.nextDouble();
            System.out.print("Enter second number: ");
            if (!sc.hasNextDouble()) {
                System.out.println("Invalid input. Please enter a number.");
                return;
            }
            double num2 = sc.nextDouble();
            double sum = num1 + num2;
            System.out.println("Sum = " + sum);
        } finally {
            sc.close();
        }
    }
}
